pipert/core/pipeline_manager.py

```diff
@@ -0,0 +1,441 @@
+import zerorpc
+import re
+from pipert.core.component import BaseComponent
+from pipert.core.errors import QueueDoesNotExist
+from pipert.core.routine import Routine
+from os import listdir
+from os.path import isfile, join
+from jsonschema import validate, ValidationError
+import functools
+
+
+# import gc
```

Comment:
delete

Reply:
I added the garbage collector module because i was'nt sure if we should force a cleaning when using the method setup_components. (at line 341 we free all of the components objects we had before)